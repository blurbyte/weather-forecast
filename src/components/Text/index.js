import styled from 'styled-components';

const Text = styled.p`
  font-size: ${props => (props.big ? '1.8rem' : '1.4rem')};
  color: #222222;
  margin: 0;
  text-transform: uppercase;
`;

export default Text;
