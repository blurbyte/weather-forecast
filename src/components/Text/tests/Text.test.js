import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Text from '../index';

test('renders correctly', () => {
  const component = renderer.create(<Text />);
  expect(component.toJSON()).toMatchSnapshot();
});
