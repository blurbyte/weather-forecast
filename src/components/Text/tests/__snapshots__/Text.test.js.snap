// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  font-size: 1.4rem;
  color: #222222;
  margin: 0;
  text-transform: uppercase;
}

<p
  className="c0"
/>
`;
